templates/guess_the_painter.html

```diff
@@ -1,37 +1,37 @@
 {% extends "layout.html" %}
 {% block body %}
 <body>
-    {% include "login_register_segment.html"%}
-    <div class="page">
-        <h1>PaintingTrivia</h1>
-        {% for message in get_flashed_messages() %}
-            <div class="flash">{{ message }}</div>
-        {% endfor %}
-        <div>Statistics:</div>
-        <div>Right guesses: {{ right_guesses }}</div>
-        <div>Wrong guesses: {{ wrong_guesses }}</div>
-        <div>Who is this painting created by?</div>
+{% include "login_register_segment.html"%}
+<div class="page">
+    <h1>PaintingTrivia</h1>
+    {% for message in get_flashed_messages() %}
+    <div class="flash">{{ message }}</div>
+    {% endfor %}
+    <div>Statistics:</div>
+    <div>Right guesses: {{ quiz.score|sum() }}</div>
+    <div>Wrong guesses: {{ quiz.score|length - quiz.score|sum() }}</div>
+    <div>Who is this painting created by?</div>
 
-        <h2>{{ selected_painting.title }}</h2>
+    <h2>{{ quiz.current_question.element.values()[0].title }}</h2>
```

Comment:
`quiz.current_question.element.values()[0].title` -- this is very ugly. when i look at it i have no idea what element, values(), [0] mean. please come up with a prettier solution